server/lib/pbench/__init__.py

```diff
@@ -238,7 +258,7 @@ def __init__(self, cfg_name):
             if not os.path.isdir(self.LIBDIR):
                 raise BadConfig("Bad LIBDIR={}".format(self.LIBDIR))  # noqa:E701
             # the scripts may use this to send status messages
-            self.mail_recipients = self.conf.get("pbench-server", "mailto")
+            self.conf.get("pbench-server", "mailto")
```

Comment:
Huh? If we're not *putting* the value anywhere, why generate it?? Or if this is just to implicitly validate that the setting exists, a better comment would help.

Reply:
Sure.